<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Concurrence</class>
 <widget class="QWidget" name="Concurrence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>通道标识</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>发送数据量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>接收数据量</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_0"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_0">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_0">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_2"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_2">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_2">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_3"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_3">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_3">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_4"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_4">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_4">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_5"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_5">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_5">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_6"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_6">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_6">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_7"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_7">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_7">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_8"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_8">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_8">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_9"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_9">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_9">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_10"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_10">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_10">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_11"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_11">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_11">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_12"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_12">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_12">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_13"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_13">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_13">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_14"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_14">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_14">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_15"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_15">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_15">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_16"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_16">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_16">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_17"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_17">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_17">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QComboBox" name="comboBoxChannel_1"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSend_1">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRecv_1">
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClear">
       <property name="text">
        <string>清零</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
